contracts/interfaces/IL2Deposit.sol

```diff
@@ -22,5 +31,7 @@ interface IL2Deposit is IStrategyBase {
         bytes depositSig;
     }
 
+    /// @notice Deposit user's fund into layer2 bridge
+    /// @param _params The deposit data following EIP-712 plus the user's signature of it
```

Comment:
`params` is a plain solidity struct. Maybe could take LimitOrder as example to refine the description.

https://github.com/consenlabs/tokenlon-contracts/blob/ece0b1215ea23b095a27e19155f2732d830a2246/contracts/interfaces/ILimitOrder.sol#L122-L133